integration-tests/tests/run-catalog-workflow.js

```diff
@@ -36,4 +31,7 @@ const testCatalog = {
   targetEnvironments: ['local', 'dev']
 }
 
-module.exports = { testCatalog }
+module.exports = {
+  linkDataToWorkspace,
```

Comment:
I would rather not have the test export a utility function. I think this confuses the purpose of the file - it becomes both test and utility. We could create a `catalog-utils.js` file for this type of code

Reply:
Ah okay! I think that's a good compromise here

Reply:
Ah good catch! Just removed it